lib/Backends/CPU/FunctionSpecializer.cpp

```diff
@@ -350,3 +355,15 @@ void LLVMIRGen::performSpecialization() {
     generateFunctionDebugInfo(&FF);
   }
 }
+
+llvm::CallInst *
+LLVMIRGen::specializeCallWithConstantArguments(llvm::CallInst *call) {
```

Comment:
Will this be used by future PRs? Doesn't seem to be called anywhere.

Reply:
Yes. Next PR will use it.